ReactSkia/components/RSkComponentImage.cpp

```diff
@@ -62,27 +62,52 @@ void RSkComponentImage::OnPaint(
     SkRect frameRect = SkRect::MakeXYWH(frame.origin.x, frame.origin.y, frame.size.width, frame.size.height);
     auto const &imageBorderMetrics=imageProps.resolveBorderMetrics(component.layoutMetrics);
     SkRect targetRect = computeTargetRect({imageData->width(),imageData->height()},frameRect,imageProps.resizeMode);
-    SkPaint paint;
-/* TO DO: Handle filter quality based of configuration. Setting Low Filter Quality as default for now*/
-    paint.setFilterQuality(DEFAULT_IMAGE_FILTER_QUALITY);
-    if(imageProps.resizeMode == ImageResizeMode::Repeat) {
-      sk_sp<SkImageFilter> imageFilter(SkImageFilters::Tile(targetRect,frameRect ,nullptr));
-      paint.setImageFilter(std::move(imageFilter));
+
+/* Draw order 1.Shadow 2. Background 3.Image Shadow 4. Image 5.Border*/
+    bool contentShadow = false;
+    bool needClipAndRestore =false;
+    if(layer()->opacity && layer()->shadowOpacity){
```

Comment:
As discussed Layer opacity is not needed to be handled in component, as it is handled in layer itself wit latest code. 
Need to consider only shadow opacity && shadow filter validity
if((layer()->shadowFilter) && (layer()->shadowOpacity) )